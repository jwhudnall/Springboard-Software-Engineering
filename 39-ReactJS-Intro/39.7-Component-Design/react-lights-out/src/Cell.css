.Cell {
  height: 100px;
  width: 100px;
  background-color: rgb(202, 199, 199);
  cursor: pointer;
}

.Cell-lit {
  background-color: rgb(91, 211, 111);
}
