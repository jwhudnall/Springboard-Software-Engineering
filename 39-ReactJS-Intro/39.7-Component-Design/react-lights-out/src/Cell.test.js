import { render, screen, fireEvent } from "@testing-library/react";
import Cell from "./Cell";

// Smoke Test
it("renders without crashing", () => {
  render(<Cell isLit={true} dataId={"0-2"} />);
});

// Snapshot Test
it("matches snapshot", () => {
  const { asFragment } = render(<Cell isLit={true} dataId={"0-2"} />);
  screen.debug();
  expect(asFragment()).toMatchSnapshot();
});
