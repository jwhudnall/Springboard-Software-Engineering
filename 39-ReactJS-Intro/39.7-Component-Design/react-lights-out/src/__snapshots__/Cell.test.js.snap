// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `<DocumentFragment />`;

exports[`switches color after switching 1`] = `<DocumentFragment />`;
